### ISSUE TITLE
Unable to create a timeline

### ISSUE THREAD

**Issue by markoa** [operately/operately#1187]
Started from a new project and no milestones, created some, then got this when clicking "Save Changes" in production:

<img width="890" alt="Screenshot 2024-09-25 at 13 27 49" src="https://github.com/user-attachments/assets/8e05f1c2-2e17-42f2-bc33-edf95773aa13">

**Comment by Rockyy174** [comment]
It seems that this issue happens when you try to save the timeline without setting a due date.

Should the user be allowed to save the timeline without a due date, @markoa?

If yes, I can update the API to handle the empty field.
If no, I can update the form to let the user know that this field can't be empty.

**Comment by markoa** [comment]
Yes, we want to allow having no due date just like when creating a project. Otherwise it becomes impossible to gradually fill in your project step by step and the system becomes too rigid.
